js/logic.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-unused-expressions */
+// eslint-disable-next-line func-names
+const game = (function () {
+  const turn = true;
+  const running = true;
+  const board = ['', '', '', '', '', '', '', '', ''];
+
+  function victory(arr) {
+    for (let i = 0; i <= 2; i += 1) {
+      if ((arr[i * 3] === arr[(i * 3) + 1]
+          && arr[i * 3] === arr[(i * 3) + 2]
+          && arr[i * 3] !== '')
+         || (arr[i] === arr[i + 3]
+          && arr[i] === arr[i + 6]
+          && arr[i] !== '')) {
+        return true;
+      }
+    }
+
+    if ((arr[0] === arr[4]
+        && arr[0] === arr[8]
+        && arr[0] !== '')
+       || (arr[2] === arr[4]
+        && arr[2] === arr[6]
+        && arr[2] !== '')) {
+      return true;
+    }
+
+    return false;
+  }
+
+  return {
+    board,
+    victory,
+    turn,
+    running,
+  };
+}());
+const player = (name) => ({
+  name,
+});
+const engine = game;
+
+const validMove = (_index, sign) => {
+  const id = dom.getId(_index);
+  if (engine.board[id] === '') {
+    engine.board[id] = sign;
+    dom.render(id, engine.board[id]);
+    return true;
+  }
+  return false;
+};
+
+let player1 = player('');
```

Comment:
Geat work using module/factory.... 👍 
But in this project we don't want to use global variable;
![image](https://user-images.githubusercontent.com/31897434/76893849-52613080-688d-11ea-9df3-a9e5fd20e243.png)